sqrl-tools/sqrl-config/src/main/java/com/datasqrl/config/SqrlConfigCommons.java

```diff
@@ -321,49 +344,83 @@ public boolean hasKey(String key) {
 
   public static SqrlConfig create(ErrorCollector errors, int version) {
     Configuration config = new BaseHierarchicalConfiguration();
-    SqrlConfigCommons newconfig = new SqrlConfigCommons(errors,null,config,"");
+    SqrlConfigCommons newconfig = new SqrlConfigCommons(errors, null, config, "");
     newconfig.setProperty(VERSION_KEY, version);
     return newconfig;
   }
 
-  public static TableConfigImpl fromFilesTableConfig(@NonNull Name name, ErrorCollector errors, @NonNull List<Path> files) {
-    return new TableConfigImpl(name, fromFiles(errors, files));
+  public static TableConfigImpl fromFilesTableConfig(@NonNull Name name, ErrorCollector errors,
+      @NonNull List<Path> files) {
+    return new TableConfigImpl(name, fromFiles(errors, Optional.of("/jsonSchema/tableConfig.json"), files));
   }
 
   public static PackageJson fromFilesPackageJson(ErrorCollector errors, @NonNull List<Path> files) {
-    return new PackageJsonImpl(fromFiles(errors, files));
+    return new PackageJsonImpl(fromFiles(errors, Optional.empty(), files));
```

Comment:
@nandorsoma Can you please add a json schema for package.json files and add it here?